File: src/Util.Platform.Share.Identity.Queries/ApplicationQueryBase.cs
namespace Util.Platform.Share.Identity.Queries;

/// <summary>
/// 应用程序查询参数
/// </summary>
public abstract class ApplicationQueryBase : QueryParameter {
    /// <summary>
    /// 应用程序编码
    ///</summary>
    [Description( "identity.application.code" )]
    public string Code { get; set; }
    /// <summary>
    /// 应用程序名称
    ///</summary>
    [Description( "identity.application.name" )]
    public string Name { get; set; }
    /// <summary>
    /// 启用
    ///</summary>
    [Description( "identity.application.enabled" )]
    public bool? Enabled { get; set; }
    /// <summary>
    /// 备注
    ///</summary>
    [Description( "identity.application.remark" )]
    public string Remark { get; set; }
    /// <summary>
    /// 起始创建时间
    /// </summary>
    [Display( Name = "util.beginCreationTime" )]
    public DateTime? BeginCreationTime { get; set; }
    /// <summary>
    /// 结束创建时间
    /// </summary>
    [Display( Name = "util.endCreationTime" )]
    public DateTime? EndCreationTime { get; set; }
    /// <summary>
    /// 起始最后修改时间
    /// </summary>
    [Display( Name = "util.beginLastModificationTime" )]
    public DateTime? BeginLastModificationTime { get; set; }
    /// <summary>
    /// 结束最后修改时间
    /// </summary>
    [Display( Name = "util.endLastModificationTime" )]
    public DateTime? EndLastModificationTime { get; set; }
}

File: src/Util.Platform.Share.Ui/ClientApp/projects/util-platform-identity/src/model/application-query-base.ts
import { QueryParameter } from "util-angular";

/**
 * 应用程序查询参数
 */
export class ApplicationQueryBase extends QueryParameter {
    /**
     * 应用程序编码
     */
    code;
    /**
     * 应用程序名称
     */
    name;
    /**
     * 启用
     */
    enabled;
    /**
    * 是否Api
    */
    isApi;
    /**
     * 备注
     */
    remark;
    /**
     * 起始创建时间
     */
    beginCreationTime;
    /**
     * 结束创建时间
     */
    endCreationTime;
    /**
     * 起始最后修改时间
     */
    beginLastModificationTime;
    /**
     * 结束最后修改时间
     */
    endLastModificationTime;
}
